pipeline/transformer/transformer_mets/src/main/scala/uk/ac/wellcome/platform/transformer/mets/service/MetsTransformerWorkerService.scala

```diff
@@ -2,23 +2,26 @@ package uk.ac.wellcome.platform.transformer.mets.service
 
 import akka.Done
 import grizzled.slf4j.Logging
-import uk.ac.wellcome.bigmessaging.{BigMessageSender, EmptyMetadata}
+import uk.ac.wellcome.bigmessaging.BigMessageSender
 import uk.ac.wellcome.json.JsonUtil._
 import uk.ac.wellcome.messaging.sns.{NotificationMessage, SNSConfig}
 import uk.ac.wellcome.messaging.sqs.SQSStream
+import uk.ac.wellcome.mets_adapter.models.MetsData
 import uk.ac.wellcome.models.work.internal.TransformedBaseWork
 import uk.ac.wellcome.platform.transformer.mets.parsers.MetsXmlParser
-import uk.ac.wellcome.storage.store.{HybridStoreEntry, VersionedStore}
-import uk.ac.wellcome.storage.{Identified, Version}
+import uk.ac.wellcome.platform.transformer.mets.store.TemporaryCredentialsStore
+import uk.ac.wellcome.storage.store.VersionedStore
+import uk.ac.wellcome.storage.{Identified, ObjectLocation, Version}
 import uk.ac.wellcome.typesafe.Runnable
 
 import scala.concurrent.Future
 
 class MetsTransformerWorkerService(
   msgStream: SQSStream[NotificationMessage],
   messageSender: BigMessageSender[SNSConfig, TransformedBaseWork],
-  store: VersionedStore[String, Int, HybridStoreEntry[String, EmptyMetadata]])
-    extends Runnable
+  adapterStore: VersionedStore[String, Int, MetsData],
+  temporaryCredentialsStore: TemporaryCredentialsStore[String]
```

Comment:
I think it would be more descriptive to name this variable according to what is stored inside it (e.g. `metsXmlStore`), rather than named after how the store accesses the data, as the data is the primary concern for this class rather than the implementation.